packages/lisk-transactions/src/utils/validation/validation.js

```diff
@@ -86,3 +86,9 @@ export const isNumberString = str => {
 	}
 	return /^[0-9]+$/g.test(str);
 };
+
+export const validateAmount = data =>
+	isNumberString(data) && !isGreaterThanMaxTransactionAmount(bignum(data));
+
+export const isValidInteger = num =>
+	typeof num === 'number' && parseInt(num, 10) === num;
```

Comment:
Is the `typeof` check necessary?

Reply:
If the num is undefined then parseInt is throwing exception. So I think we need it. Also proper polyfill is here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger#Polyfill